prow/plugins/label/label.go

```diff
@@ -186,6 +188,12 @@ func handle(gc githubClient, log *logrus.Entry, additionalLabels []string, e *gi
 		}
 	}
 
+	for _, customLabel := range customLabels {
```

Comment:
nit: this can be

```go
nonexistent = RepoLabelsExisting.Intersection(sets.NewString(customLabels...)).List()
```

(or just use sets instead of slices, etc)

Reply:
While implementing this, I actually realized that this part was redundant. In the `getLabelsFromGenericMatches()` call, custom labels that are not included in `additionalLabels` are already being added to `nonexistent`. The issue that was happening was simply that other non-custom labels were being added to the slice, and that has now been removed. However, in the following stanza, we are ignoring labels that are part of `additionalLabels` but are not present on the repo. This was already occurring, but just wanted to check that we do in fact want this behavior:

```go
if !RepoLabelsExisting.Has(labelToAdd) {
    continue
}
```

We now do not post a comment in this scenario.